### DISCUSSION TITLE
Error messages should have aria-live="polite" included

### DISCUSSION THREAD

**Discussion by BennyAlex** [lookfirst/mui-rff#1174] [Ideas]
For screenreader users its hard to tell when an field has an error.
By adding aria-live="polite" it will be announced to them.

**Comment by lookfirst** [comment]
Hey, thanks for the issue. I do have to say that this is likely something we won't add. It is something that you should get MUI to add and we would just pass those props through. We are just a middle layer between MUI and RFF to enable form handling to work properly.

**Reply by BennyAlex** [reply]
Ah I thought your already using the components seperatly like https://mui.com/material-ui/react-text-field/#components

If you use only the textfield its harder, true. I can ask there sure.
